tests/integration/test_storage_rabbitmq/test.py

```diff
@@ -274,7 +275,7 @@ def test_rabbitmq_tsv_with_delimiter(rabbitmq_cluster):
         channel.basic_publish(exchange='tsv', routing_key='', body=message)
 
     connection.close()
-    time.sleep(1)
+    time.sleep(2)
```

Comment:
Most likely it will make test less flaky, but will not fix anything, because tests with `sleep` are usually unreliable.